content/docs/command-reference/remote/modify.md

```diff
@@ -333,14 +333,21 @@ following parameters (listed in order or precedence) to customize the
 authentication method:
 
 1. `connection_string` is used for authentication if given (all others params
-   are ignored).
+   are ignored). If `connection_string` is not set, the `account_name` must be
+   given.
 2. If `tenant_id` and `client_id` or `client_secret` are given, Active Directory
    (AD)
    [service principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
    auth is performed.
-3. The storage `account_name` is tried next, along with `account_key` or
-   `sas_token` (in that order). If neither key nor token is provided, DVC will
-   try to connect anonymously.
+3. If either `account_key` or `sas_token` (in this order) are set, then DVC will
+   try to connect with those options.
```

Comment:
```suggestion
3. DVC will try next to connect with `account_key` or `sas_token` (in this
   order) if either are provided.
```

Reply:
Thanks, good copy edit. We can usually just commit these BTW (we don't expect the core team to write perfectly). BUT I have a Q on this (https://github.com/iterative/dvc.org/pull/2489#pullrequestreview-665102806) so let's wait a bit...

Reply:
that comment is marked as resolved?

Reply:
> that comment is marked as resolved?

Yeah, @isidentical answered the Q so he resolved it. I unresolved it now for visibility.

Anyway, looks like we can just commit this...